Configurations/ConfigTestApp.xcconfig

```diff
@@ -8,4 +8,3 @@ PRODUCT_BUNDLE_IDENTIFIER = org.sparkle-project.SparkleTestApp
 CODE_SIGN_ENTITLEMENTS = TestApplication/Sparkle-Test-App.entitlements
 MACOSX_DEPLOYMENT_TARGET = 10.10
```

Comment:
```suggestion
MACOSX_DEPLOYMENT_TARGET = 10.11
```

Reply:
Good catch. We had several components already with 10.10 or 10.11 min requirement outside of the framework (unit tests, test app, sparkle-cli, ..). I just removed deployment target on all of these and they now inherit 10.11 from the common config.

Reply:
Excellent decision 👍🏻